README.rst

```diff
@@ -20,7 +20,7 @@ Dependencies
 ------------
 
 * Python 2.6+ (not Python 3)
```

Comment:
Since I've been here, we've never tested against Python 2.6. Is that something we need to do?